### ISSUE TITLE
GeoBase initiative page - Single Page communities Design

### ISSUE THREAD

**Issue by sean-eagles** [Canadian-Geospatial-Platform/geo.ca#108]
The single page design is what we would like to have built as a template for community pages.  Vlad will begin by engaging staff and following through the design phase.

**Comment by vgulyy** [comment]
This is now with Jared King for the WP implementation.

**Comment by jaredkinger** [comment]
This has been implemented and is up on https://wordpres.dev.geo.ca/communities/geobase/

Currently waiting for updated content from Cindy. We also require French translations.

@sean-eagles Are we targeting the February release for this? There is currently no release assigned to this.

**Comment by elatsis** [comment]
> This has been implemented and is up on https://wordpres.dev.geo.ca/communities/geobase/
> 
> Currently waiting for updated content from Cindy. We also require French translations.
> 
> @sean-eagles Are we targeting the February release for this? There is currently no release assigned to this.

@jaredkinger this will be included in the June release - just updated it

**Comment by jaredkinger** [comment]
The initial design included a section titled "Elsewhere on GEO,ca". This section featured several links to additional GeoBase pages however since we condensed all of the GeoBase information down to a single page those additional pages no longer exist. This section has been renamed "GeoBase Resources" and will list any links to external websites or documents that are relevant to the community. Currently this only lists the CCOG.

It has been requested to add community announcements to the page. Ideally these would pull from the geocore API's `/community/announcements/get` endpoint and would update dynamically but that information is not currently loaded into geocore. 

I've updated the "Elsewhere" section to "Resources" and added a placeholder for announcements here: https://wordpres.dev.geo.ca/communities/geobase/. Once we have a more concrete example of what an actual announcement's content will be @vgulyy may make some adjustments to the announcement styling.

We've reached out to Cindy to get content for both the Resources and Announcements.
